docs/semgrep-cloud-platform/user-management.md

```diff
@@ -112,7 +112,7 @@ The following table displays features available to each role:
 | Projects              | no        | yes       | Only `admin` can manage projects.                                                  |
 | Rule Board (Policies) | no        | yes       | Only `admin` can manage policies and rules.                                        |
 | Findings              | yes       | yes       | Both `admin` and `member` roles can sort, filter, comment on, and triage findings. |
-| Editor                | yes       | yes       | `member` roles are able to write and modify rules within an organization.          |
+| Editor                | yes       | yes       | `member` access is read-only within their organization. Users with a `member` role can use their personal account to write a rule.  |
```

Comment:
@s-santillan I'd exclude the `Users with a `member` role can use their personal account to write a rule.`

This note basically describes a work around - but it may not always be valid, or at least is something we can handle suggesting on an individual basis.